src/utils/VariableVisitor.js

```diff
@@ -1,18 +1,44 @@
 import titleGenerator from './TitleGenerator';
-
+/**
+ * Class to visit the variables using the ciceromark of a template.
+ */
 class VariableVisitor {
+  /**
+   * Visit the children.
+   *
+   * @param {Function} visitor    Visit the variable
+   * @param {object}   thing      CiceroMark object of a template
+   * @param {object}   parameters Various functions for an object
+   * @param {Array}    result     Result array
+   * @param {string}   field      Field name
+   */
   static visitChildren(visitor, thing, parameters, result, field = 'nodes') {
     if(thing[field]) {
       VariableVisitor.visitNodes(visitor, thing[field], parameters, result);
     }
   }
 
+  /**
+   * Visit the nodes for fields other than Variable type.
+   *
+   * @param {string} visitor    Type of VariableVisitor
+   * @param {object} things     CiceroMark JSON for the field
+   * @param {object} parameters Counter for variables
+   * @param {Array}  result     Variable titles
+   */
   static visitNodes(visitor, things, parameters, result) {
     things.forEach(node => {
       visitor.visit(node, parameters, result);
     });
   }
 
+  /**
+   * Update the counters for variable fields and visit the sub fields for other types.
```

Comment:
Write the verbs in third-person form as the majority of the docstrings you have written are in the same form.